charts/error-pages/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Kubernetes error pages for traefik
 name: error-pages
-version: 0.1.2
+version: 0.1.3
```

Comment:
pls update the version to 1.0.0 as this is a breaking change

Reply:
Hey @rpahli , thanks, have done so.